modules/pam_faillock/main.c

```diff
@@ -123,8 +126,80 @@ args_parse(int argc, char **argv, struct options *opts)
 static void
 usage(const char *progname)
 {
-	fprintf(stderr, _("Usage: %s [--dir /path/to/tally-directory] [--user username] [--reset]\n"),
-		progname);
+	fprintf(stderr,
+		_("Usage: %s [--dir /path/to/tally-directory] "
+		  " [--user username] [--reset] [--legacy-output]\n"), progname);
+
+}
+
+static int get_local_time(time_t when, char *timebuf, unsigned long timebuf_size)
+{
+	struct tm *tm;
+
+	tm = localtime(&when);
+	if(tm == NULL) {
```

Comment:
Even if `if(tm == NULL) ` comes from an old code, the absence of space after `if` looks odd here.